uportal-war/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp

```diff
@@ -273,6 +273,7 @@
         </a>
     </li>
     <li class="divider"></li>
+    {% if (canFavorite) { %}
```

Comment:
It's hard to say in this case.  This UI is build with Underscore, which was briefly popular for portlet development some time ago.  I don't know it well, and I'd say most of the folks who might need to maintain this page don't know it well.

But I don't have time to rewrite it.  I went with the Underscore-style conditional because this template has loads more Underscore-style conditional and looping right beside it.  I din't want to make things more confusing by introducing yet another syntax within the same section.